src/topp/IDFileConverter.cpp

```diff
@@ -170,6 +170,7 @@ class TOPPIDFileConverter :
     registerFlag_("ignore_proteins_per_peptide", "[Sequest only] Workaround to deal with .out files that contain e.g. \"+1\" in references column,\n"
                                                  "but do not list extra references in subsequent lines (try -debug 3 or 4)", true);
     registerStringOption_("scan_regex", "<expression>", "", "[Mascot, pepXML, Percolator only] Regular expression used to extract the scan number or retention time. See documentation for details.", false, true);
+    registerFlag_("no_spectra_data_override", "[+mz_file only] Will switch off spectra_data override in ProteinIdentifications if mz_file is given and spectrum_references are added/updated. Switch off only if you are sure it is absolutely the same mz_file as used for identification.", true);
```

Comment:
spectra_data and spectrum_references are meta params, which means they are inspectable by looking in the file _and_ they are "explicitly" not required for a valid file but might be essential for building a functioning workflow. And I think we have to distinguish between the regular user and the advanced user, who is likely to look into the files and the advanced parameter set of tools.